README.md

```diff
@@ -203,7 +203,6 @@ the following:
 ```shell
 $ cat .bazelrc
 build --workspace_status_command=./print-workspace-status.sh
-run --workspace_status_command=./print-workspace-status.sh
```

Comment:
Why did you remove this?

Reply:
See commit message:
Simplify .bazelrc example.  …
According to
https://docs.bazel.build/versions/master/command-line-reference.html#run
run inherits from build and therefore we can omit the entry for run.